snazzy gives incorrect message for "Missing '()' invoking a constructor"

I've been all over the place with this issue, posting on [standard](https://github.com/feross/standard/issues/725), then [ESLint](https://github.com/eslint/eslint/issues/7792), back to standard and now here 😆 

If I have the code:

```js
const onionoo = new Onionoo()
```

to create a new instance of the `Onionoo` class but leave off the brackets:

```js
const onionoo = new Onionoo
```

`standard` and ESLint give the error `Missing '()' invoking a constructor` but `snazzy` just returns `Missing '`:

```
$ standard
standard: Use JavaScript Standard Style (http://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.
  /Users/lukechilds/Dev/oss/onionoo-node-client/test/unit.js:14:19: Missing '()' invoking a constructor.

$ snazzy
standard: Use JavaScript Standard Style (http://standardjs.com)
standard: Run `standard --fix` to automatically fix some problems.

/Users/lukechilds/Dev/oss/onionoo-node-client/test/unit.js
  14:19  error  Missing '

✖ 1 problem
```

--- a/index.js
+++ b/index.js
@@ -8,7 +8,7 @@ function jsonify (rawtext, opts) {
   var results = []
   var resultMap = {}
   lines.forEach(function (line) {
-    var re = /\s*([^:]+):([^:]+):([^:]+): ([^(]*) ?(\((.*)\))?/.exec(line)
+    var re = /\s*([^:]+):([^:]+):([^:]+): (.*?)( \((.*)\))?$/.exec(line)
     if (!re) return opts.noisey ? console.error(line) : null
 
     var filePath = re[1]

--- a/test/index.js
+++ b/test/index.js
@@ -10,7 +10,7 @@ test('standard-json', function (t) {
 
   var output = jsonify(data)
 
-  t.deepEqual(dataJson, output, 'JSON formatted')
+  t.deepEqual(output, dataJson, 'JSON formatted')
 })
 
 test('standard-json verbose', function (t) {
@@ -20,5 +20,5 @@ test('standard-json verbose', function (t) {
 
   var output = jsonify(data)
 
-  t.deepEqual(dataJson, output, 'JSON formatted')
+  t.deepEqual(output, dataJson, 'JSON formatted')
 })

